webapp/src/components/content/dividerElement.tsx

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react'
+
+import {MutableDividerBlock} from '../../blocks/dividerBlock'
```

Comment:
this is still outside this file, I think is ok for now, and maybe forever, but something to think about.